jest.config.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
   "snapshotSerializers": ["enzyme-to-json/serializer"],
   "setupFilesAfterEnv": ["<rootDir>/setupEnzyme.ts"],
   "moduleNameMapper": {
-    ".+\\.(css|scss)$": "<rootDir>/styleMock.ts"
+    ".+\\.(css|scss)$": "<rootDir>/styleMock.ts",
```

Comment:
to avoid polluting the project's root, move these mock modules to a subdirectory